<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TransformColumnWidget</class>
 <widget class="QWidget" name="TransformColumnWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>467</width>
    <height>261</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <spacer name="_ignore_horizontal_spacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>126</width>
       <height>37</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QWidget" name="frame_widget" native="true">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QFormLayout" name="_ignore_form_layout">
        <item row="0" column="0">
         <widget class="QLabel" name="_ignore_label">
          <property name="text">
           <string>Column</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="cb_column_list"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="_ignore_label_2">
          <property name="text">
           <string>Data Type</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLabel" name="lb_data_type">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QRadioButton" name="rb_expr">
          <property name="text">
           <string>Transform With Formula</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="4" column="0" colspan="2">
         <widget class="LineEdit" name="le_formula_editor">
          <property name="placeholderText">
           <string>Enter formula here!</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QRadioButton" name="rb_auto_increament">
          <property name="text">
           <string>Auto increament wih initial value</string>
          </property>
         </widget>
        </item>
        <item row="6" column="0" colspan="2">
         <widget class="QLineEdit" name="le_init_value">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="placeholderText">
           <string>Enter inital value here!</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="_ignore_horizontal_layout">
        <item>
         <spacer name="_ignore_horizontal_spacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="btn_transform">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Transform</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="_ignore_vertical_spacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="_ignore_horizontal_spacer_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>126</width>
       <height>38</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LineEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">LineEdit.hpp</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>rb_expr</sender>
   <signal>toggled(bool)</signal>
   <receiver>le_formula_editor</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>147</x>
     <y>87</y>
    </hint>
    <hint type="destinationlabel">
     <x>233</x>
     <y>116</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rb_auto_increament</sender>
   <signal>toggled(bool)</signal>
   <receiver>le_init_value</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>194</x>
     <y>145</y>
    </hint>
    <hint type="destinationlabel">
     <x>233</x>
     <y>174</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
